multivac/gather_job_data.py

```diff
@@ -192,21 +211,20 @@ def put_to_db(self):
             }
             if 'runner_version' in curr_job.keys():
                 tags['runner_version'] = curr_job['runner_version']
+
             if 'runner_name' in curr_job.keys():
                 tags['runner_name'] = curr_job['runner_name']
-            if 'failure_type' in curr_job.keys():
-                tags['failure_type'] = curr_job['failure_type']
 
             fields = {
-                'started_at': curr_job['started_at'],
-                'completed_at': curr_job['completed_at'],
+                'time_in_queue_sec': time_job_in_queue,
+                'time_in_progress_sec': time_job_in_progress,
             }
 
             data = {
-                'measurement': f'Job {job}',
+                'measurement': measurement,
                 'tags': tags,
                 'fields': fields,
-                'time': unix_ns_time
+                'time': int(time_job_queued * 1000000000)
```

Comment:
looks like we can lose precision in this line

Reply:
what's in this magic number? why 10^9?

Reply:
Earlier we get `time_job_queued` in seconds, here we transfer it to nanoseconds. I'll make a comment